spec/setup_acceptance_node.pp

```diff
@@ -7,7 +7,7 @@
 }
 
 # Defaults to staging, for release, use
```

Comment:
I think we initially did this so we could workaround the staleness of Katello's repos. IMHO we should use `candlepin::repo` once the nightly packages are green again.